### PR TITLE
Wdfn 150 do not show graph for no data sites

### PR DESCRIPTION
**Pull request by ayan-usgs**
I also tried dealing with not showing the graph or any of its peripheral controls by not appending any thing to div if there aren't timeseries, but that got real messy. I opted to hide the plot by adjusting the styling.

### CODE REVIEW

File: assets/src/scripts/components/hydrograph/index.js [hunk 1]
```diff
@@ -296,9 +296,31 @@ const createTitle = function(elem) {
...
         }, titleSelector));
 };
+/**
+ * Modify styling to hide or display the plot area.
+ *
+ * @param elem
+ * @param allTimeseries
+ */
+const controlGraphDisplay = function (elem, allTimeseries) {
+    const seriesWithPoints = Object.values(allTimeseries).filter(x => x.points.length > 0 && x.tsKey === 'current');
```
Comment (mbucknell) [inline assets/src/scripts/components/hydrograph/index.js]: You can at least partially filter the time series with the timeSeriesSelector('current') (in the timeseries module).

File: waterdata/templates/macros/components.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% macro TimeseriesComponent(site_no) -%}
     <div class="wdfn-component" data-component="hydrograph" data-siteno="{{ site_no }}">
-        <div class="compare-container" data-siteno="{{ site_no }}">
+        <div class="compare-container" style="display: none;" data-siteno="{{ site_no }}">
```
Comment (mbucknell) [inline waterdata/templates/macros/components.html]: I wonder if using the "hidden" attribute would work? That's the recommendation from the designsystem.digital.gov website.

**Comment by ayan-usgs** [comment]
> check to see if the hidden attribute would work

`hidden` works pretty well.

File: assets/src/scripts/components/hydrograph/index.js (hunk 1 above)
Comment (ayan-usgs) [reply assets/src/scripts/components/hydrograph/index.js]: Good idea! There are so many different selectors, I sometimes lose track of them. :flushed:

**Pull request merged**